js/bullets.js

```diff
@@ -1,25 +1,28 @@
 var BulletForce = 5;
 
-var Bullet = function(x, y, up){
-  this.pos = createVector(x, y);
+var Bullet = function(x, y, up) {
+    this.pos = createVector(x, y);
 
-  //Creates acceleration upwards or downwards, if up
-  this.acc = createVector(0, BulletForce * (up ? -1 : +1 ) );
+    //Creates acceleration upwards or downwards, if up
+    this.acc = createVector(0, BulletForce * (up
+        ? -1
```

Comment:
This is bad coding style and can cause difficulty in reading as the operation is spread to 3 lines